tests/sentry/eventstore/snuba/test_backend.py

```diff
@@ -67,15 +67,16 @@ def setUp(self):
 
     def test_get_events(self):
         events = self.eventstore.get_events(
-            filter=Filter(project_ids=[self.project1.id, self.project2.id])
+            filter=Filter(
+                project_ids=[self.project1.id, self.project2.id],
+                conditions=[["type", "!=", "transaction"]],
```

Comment:
Should we leave a todo here to remove this once we can no longer have transactions here?